peer/peer.go

```diff
@@ -450,7 +451,7 @@ type Peer struct {
 
 	wireEncoding wire.MessageEncoding
 
-	knownInventory     *mruInventoryMap
+	knownInventory     lru.Cache
```

Comment:
Could this have been a pointer (i.e. *lru.Cache)?

Reply:
Not really useful as it's not being passed around anywhere.